pkg/simple/client/monitoring/prometheus/metrics_rules.go

```diff
@@ -0,0 +1,418 @@
+/*
+Copyright 2019 The KubeSphere Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package prometheus
+
+import (
+	"fmt"
+	"kubesphere.io/kubesphere/pkg/simple/client/monitoring"
+	"strings"
+)
+
+const (
+	StatefulSet = "StatefulSet"
+	DaemonSet   = "DaemonSet"
+	Deployment  = "Deployment"
+)
+
+var metricsPromqlMap = map[monitoring.MetricsLevel]map[string]string{
+
+	monitoring.MetricsLevelCluster: {
```

Comment:
Can we use a configuration file and load them at startup, since these are static rules, using configuration would make it easy to change.